src/mame/drivers/itech32.cpp

```diff
@@ -1011,8 +1012,7 @@ void itech32_state::itech020_map(address_map &map)
 	map(0x500000, 0x5000ff).rw(FUNC(itech32_state::bloodstm_video_r), FUNC(itech32_state::bloodstm_video_w));
 	map(0x578000, 0x57ffff).nopr();             /* touched by protection */
 	map(0x580000, 0x59ffff).ram().w(m_palette, FUNC(palette_device::write32)).share("palette");
-	map(0x600000, 0x603fff).ram().share("nvram32");
-/* ? */ map(0x61ff00, 0x61ffff).nopw();            /* Unknown Writes */
+	map(0x600000, 0x61ffff).ram().share("nvram32");
 	map(0x680002, 0x680002).r(FUNC(itech32_state::itech020_prot_result_r));
```

Comment:
Do we have proof that ALL itech020 boards had 128k of NVRAM? If not, then this change should only apply to pubball, which already had its own map.

Reply:
All? No. Do we have proof WHICH itech020 games had 128k of NVRAM? Also no.

But the fact that the back end of the 128k range had enough writes to merit a nopw() handler would indicate that it was common enough, at least.

I specifically advised that Travis make this change, and I stand by it.